nixos/modules/system/boot/stage-1.nix

```diff
@@ -126,6 +126,10 @@ let
       ${optionalString (any (fs: fs.autoResize) fileSystems) ''
         # We need mke2fs in the initrd.
         copy_bin_and_libs ${pkgs.e2fsprogs}/sbin/resize2fs
+        # Copy also f2fs-tools' fsck and resize
+        # TODO: separate these in case no f2fs fs are present
```

Comment:
Can you add this check?

Reply:
I thought about it, but I don't seem to understand how this conditional works: `(any (fs: fs.autoResize) fileSystems)`

If you can explain it or link me to documentation how this works, I'd be happy to implement separate conditions for ext* and f2fs resizing like this:
```
if anyfs.type("f2fs").hasparam(autoresize):
    # include f2fs-tools
if anyfs.type("ext*").hasparam(autoresize):
    # include e2fsprogs
```

Reply:
We have a similar check for zfs: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/tasks/filesystems/zfs.nix#L18

Reply:
or better this one: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/tasks/filesystems/zfs.nix#L51

Reply:
@Mic92 I pushed a separate check for f2fs. The only problem is - it includes mke2fs even if no ext* filesystems need resizing, because I wasn't sure how to wildcard ext2, ext3 and ext4 filesystems in the check.